data/dfdevaney.json

```diff
@@ -15,7 +15,17 @@
     {
       "name": "LinkedIn",
       "url": "https://www.linkedin.com/in/doug-devaney",
-      "icon": "FaLinkdedIn"
+      "icon": "FaLinkedIn"
+    },
+    {
+      "name": "Threads",
+      "url": "https://www.threads.net/@Snoops127",
+      "icon": "FaThreads"
```

Comment:
Hi, "FaThreads" is not a valid icon. Consider searching for available icons [here](https://linkfree.io/icons)
![icon](https://github.com/EddieHubCommunity/LinkFree/assets/19821445/67471eba-d1fe-40e0-83f0-462f537b3ae0)

```suggestion

```

Reply:
Thanks!
I was wondering where the icon list was located.
I will need to make an adjustment later.

Reply:
Icons not found will go to a default icon. It is best to keep the property and not delete this icon line altogether.

I will merge this PR and once you wish to make the adjustment @dfdevaney please raise a new PR.